tests/testthat/test-derive_merged.R

```diff
@@ -169,8 +193,36 @@ test_that("derive_vars_merged Test 6: expressions for new_vars and missing_value
   )
 })
 
-## Test 7: use new variables in filter_add and order ----
-test_that("derive_vars_merged Test 7: use new variables in filter_add and order", {
+
+## Test 8: expressions for new_vars and missing_values and exist_flags ----
+test_that("derive_vars_merged Test 8: expressions for new_vars and missing_values and exist_flags", {
+  actual <- derive_vars_merged(
+    adsl,
+    dataset_add = advs,
+    by_vars = exprs(USUBJID),
+    order = exprs(AVISIT),
+    new_vars = exprs(LASTVIS = str_to_upper(AVISIT)),
+    mode = "last",
+    missing_values = exprs(LASTVIS = "UNKNOWN"),
```

Comment:
@bundfussr the modifications to `derive_vars_merged()` to handle the use of `missing_values` and `exist_flag` are reflected in this test. It actually made more sense to me to handle them separately.